lib/src/voip/backend/livekit_backend.dart

```diff
@@ -336,21 +347,26 @@ class LiveKitBackend extends CallBackend {
 
     final callId = keyContent.callId;
 
+    final p =
+        CallParticipant(groupCall.voip, userId: userId, deviceId: deviceId);
+
     if (keyContent.keys.isEmpty) {
       Logs().w(
           '[VOIP E2EE] Received m.call.encryption_keys where keys is empty: callId=$callId');
       return;
     } else {
       Logs().i(
-          '[VOIP E2EE]: onCallEncryption, got keys from $userId:$deviceId ${keyContent.toJson()}');
+          '[VOIP E2EE]: onCallEncryption, got keys from ${p.id} ${keyContent.toJson()}');
     }
 
+    requestEncrytionKeyPending.remove(p)?.dispose();
```

Comment:
How can you be sure, that you are not disposing of the wrong request here? I.e. you have:

1. undecryptable frame
2. request keys
3. stream change
4. request keys
5. answer for 2.
6. cancel request for 4.

Does that not matter, since the _setEncryptionKey will fire another undecryptable event?

Reply:
moved to https://github.com/famedly/product-management/issues/2078